JudoPay.tsx

```diff
@@ -32,7 +38,26 @@ class JudoPay {
         });
     }
 
-    private generateParameters(
+    public invokePaymentMethodScreen(
+        mode: JudoTransactionMode,
+        configuration: JudoConfiguration
+    ): Promise<JudoResponse | null> {
```

Comment:
`Promise<JudoResponse | null>` can we just `Promise<JudoResponse>` ?

Reply:
It can be null if response won't be that type as expected.

Reply:
@AndreiGalkin is it possible to throw an error in case of `response won't be that type as expected` ?

Reply:
I think yes, check for type similarity while parsing or wrap in exception handler